src/leiningen/javadoc.clj

```diff
@@ -25,6 +46,8 @@
      :package-names (get javadoc-opts :package-names)
      :additional-args (get javadoc-opts :additional-args)
      :exact-command-line (get javadoc-opts :exact-command-line)
+     :jdk-home (get javadoc-opts :jdk-home)
+     :java-cmd (get javadoc-opts :java-cmd)
```

Comment:
I'd also be happy to add a commit that compresses this into a select-keys + a merge.